assets/authchoice.js

```diff
@@ -28,7 +28,7 @@ jQuery(function($) {
         return this.each(function() {
             var $container = $(this);
 
-            $container.find('a').on('click', function(e) {
+            $container.find('a.auth-link').on('click', function(e) {
```

Comment:
But if someone is already messing with the CSS classes, they can just add `auth-link` to manually.  This specific change would at least be be a non-breaking change that would still work for all existing widgets while allowing a certain level of control if needed.